src/components/ContributePrompt.js

```diff
@@ -111,14 +112,19 @@ export default class ContributePrompt extends Component {
   render() {
     const { details, summary, show, resolution } = this.state
     const { session, onLogin } = this.props
-
     return (
       <Modal show={show} id="contribute-modal">
         <Form>
           <Modal.Header closeButton onHide={this.close}>
             <Modal.Title>Describe the changes in this curation</Modal.Title>
           </Modal.Header>
           <Modal.Body>
+            {!session.isAnonymous && !session.publicEmails && (
+              <Alert bsStyle="warning">
+                Since you set your email as private on your GitHub profile, you can submit this contribution but the
+                commits would not be attributed to you.
```

Comment:
Perhaps: Since your email is set as private on your GitHub profile, you can submit this contribution but the
                commits will not be attributed to you.